df_py/predictoor/csvs.py

```diff
@@ -8,6 +11,7 @@
 from df_py.util.csv_helpers import assertIsEthAddr
 
 
+# ------------------------------- PREDICTOOR DATA -------------------------------
 @enforce_types
```

Comment:
Predictoor is on only one chain, for the foreseeable future. Therefore you don't need to specify chainid in the filename. Instead the filename can simply be "predictoor_data.csv". Similar to Challenge DF.

Currently this cluster of functions for predictoor csv data is around having >1 file. With only one file, it can be simpler. You should (heavily) revise it, patterning after challenge's csvs.py. 

So the functions here are simply:
```python
def save_predictoor_data_csv(predictoor_data, csv_dir: str)
def load_predictoor_data_csv(csv_dir: str) -> Tuple[List[str], List[str], list]:
def predictoor_data_csv_filename(csv_dir: str) -> str:
```